generate.sh

```diff
@@ -38,7 +38,7 @@ function compile() {
     --doc_out=json,generated.json:. $proto_files
   popd
   
-  ./merger $COMPONENT
+  ./merger $COMPONENT "$REPO_URL" "$PROTO_SRC_DIR" "$COMMIT"
```

Comment:
I also needed these values, in addition to five more, in order to generate the daemon summary pages (ex: https://staging.lightning.engineering/api-docs/docs/api/lnd/)

In my PR, I used the approach of capturing the environment vars instead of passing in so many args via command line.
https://github.com/lightninglabs/lightning-api-ng/pull/2/commits/c051e0df614398bb3c54f6f492d6f351094e5ba4

I can adapt my PR to your approach if you don't think it's a problem to pass in the 8 args to `merger`.

Reply:
Great idea! Added your changes as a commit (and the Taro ones too). So these variables are now rendered to the JSON (at the bottom).